ckanext/hierarchy/templates/organization/snippets/organization_tree.html

```diff
@@ -14,7 +14,7 @@
 
 #}
 
-{% set type = 'asset' if h.ckan_version() > '2.9' else 'resource' %}
+{% set type = 'asset' if h.check_ckan_version(min_version='2.10') else 'resource' %}
```

Comment:
We want to use `asset` on CKAN 2.9 as well right (existing logic was wrong, should had been `>= 2.9`)? So this should be:

```suggestion
{% set type = 'asset' if h.check_ckan_version(min_version='2.9') else 'resource' %}
```

Reply:
True, dug through the docs and `asset` is indeed used from CKAN 2.9, whereas `resource` is used up until CKAN 2.8. Thanks!